rust/src/delta.rs

```diff
@@ -1116,52 +1103,13 @@ pub enum DeltaTransactionError {
     },
 }
 
-/// Error that occurs when a single transaction commit attempt fails
-#[derive(thiserror::Error, Debug)]
-pub enum TransactionCommitAttemptError {
```

Comment:
I have removed this as it was not different to `DeltaTransactionError` at all and introduced additional mapping/layers